Cargo.toml

```diff
@@ -12,17 +12,18 @@ categories = ["embedded", "no-std"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[dependencies]
-postcard = "0.4.2"
-heapless = "0.5.1"
+[dependencies.postcard]
+version = "0.7.0"
+default-features = false
 
 [dependencies.serde]
-version = "1.0.102"
+version = "1.0.126"
 default-features = false
 
 [dev-dependencies]
 hex = "0.4.0"
 
 [features]
 default = ["std"]
-std = ["serde/std"]
+std = ["serde/std", "postcard/use-std"]
+no_std = ["postcard/heapless-cas"]
```

Comment:
```suggestion
```

Reply:
Thanks for the PR!  I've checked my implementation and I believe that:
1. I was not using the `heapless` crate (so thanks for removing it from the dependencies!)
2. I'm not using any function enabled by the "heapless-cas" feature of postcard, so my suggestion is to remove the "no_std" feature you introduced, so that it's either "std" or no-feature.

I made the implementation so that the urpc server can be `std` an `no_std`, where the no_std version doesn't require any feature.  The server just uses slice references, so there's no need for `heapless` there or any vector type.

The urpc client is implemented only in `std` and uses vectors, so right now it's not suitable for `no_std`.

Let me know if that makes sense, and if you agree, please update the PR removing the suggested line :)